R/make_datatagr.R

```diff
@@ -119,14 +119,13 @@ make_datatagr <- function(x,
   args <- rlang::list2(...)
 
   if (length(args) && is.list(args[[1]])) {
-    warning(
-      "The use of a list of tags is deprecated. ",
+    stop(
```

Comment:
That is true - it was my initial take as well. After removing this, the function would spit out a datatagr without any warning or issue so it would look like it succeeded. This is when I (re)introduced this throw.

I am happy to remove the code as it is exceptionally specific and the docs clearly state what works. If this becomes an issue down the line it is quickly reintroduced.